src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -121,8 +121,14 @@ export default class PickEntityOrConversionDialog extends CommonMixin {
 
   showSocietiesInfo = false
 
-  @Watch('showModal')
-  handleModalClose (newVal) {
+  closeIconClicked () {
+    this.setPickEntityModalVisible(false)
+    // clear "View all business types" selection
+    this.setEntityTypeCd(null)
```

Comment:
Compare this to search.vue where we set this to empty string. This maintains reactivity, so the v-select clears itself correctly.